src/app/statistic/statistic.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'sh-statistic',
```

Comment:
Then what is the difference between this component and dashboard? If it's just a list of sensors then name of the component should be changed because it confuses so much.